packages/pictograms/pictograms.yml

```
@@ -110,6 +110,21 @@
     - sow
     - hand
     - soil
+- name: ahmadabad--jama-masiid
```

The new entry should go between the "agile" and "agility" entries to maintain alphabetical order.

```suggestion
- name: agile
  friendly_name: Agile
  aliases:
    - agile
- name: agile--coaching
  friendly_name: Agile coaching
  aliases:
    - agile
    - coaching
    - mentoring   
- name: agility
  friendly_name: Agility
  aliases:
    - agility
    - cloud
- name: agility--02
  friendly_name: Agility 02
  aliases:
    - agility
    - cloud
    - agile
    - scrum
- name: agility-with-hybrid-multicloud
  friendly_name: Agility with hybrid multicloud
  aliases:
    - agility with hybrid multicloud
- name: agriculture
  friendly_name: Agriculture
  aliases:
    - agriculture
    - farm
    - plant
    - seeds
    - grow
    - sow
    - hand
    - soil
- name: ahmadabad--jama-masiid
```